target/linux/rtl838x/base-files/lib/preinit/07_set_preinit_iface_rtl838x

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
+# Copyright (C) 2020 OpenWrt.org
+#
```

Comment:
preinit are library files, so should be non-executable and thus not require a shebang (and remove the copyright as stated above).

Reply:
I removed the file. It was a hack, see below.